dbms/src/Common/HashTable/Hash.h

```diff
@@ -65,7 +65,7 @@ inline DB::UInt64 intHash64(DB::UInt64 x)
 #include <arm_neon.h>
 #endif
 
-inline DB::UInt64 intHashCRC32(DB::UInt64 x)
+inline DB::UInt32 intHashCRC32(DB::UInt64 x)
```

Comment:
Will this increase the hash collision?

Reply:
No. crc32 function will always return UInt32. For other platforms, UInt64 returned by `intHash64` is converted to a UInt32 by XORing the first 32 bits with the last 32 bits.